lib/rubocop/cop/style/empty_block_parameter.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for delimiters that are pipes or braces for empty
+      # block parameters. Delimiters for empty block parameters do not cause
+      # syntax errors, but it looks strange.
```

Comment:
"It looks strange" is a subjective valuation, I guess. Perhaps "are redundant"? 🙂